### DISCUSSION TITLE
Hono deleting my folder(I am new to hono)

### DISCUSSION THREAD

**Discussion by 26thavenue** [honojs/hono#2536] [Q&A] [answered]
So every time I clone my empty repo from github and try to do bunx create hono I get a message saying the file is not empty despite there only being the invisible .git file and if i select yes it gives an error -  "  export default "/tmp/bunx-1000-create-hono@latest/node_modules/create-hono/bin";
                ^
error: could not find commit hash for main
 code: "MISSING_REF"

      at new DegitError (/tmp/bunx-1000-create-hono@latest/node_modules/create-hono/bin:17108:9)
      at /tmp/bunx-1000-create-hono@latest/node_modules/create-hono/bin:17473:17"
and deletes my folder . How can I overcome this

**Comment by 26thavenue** [comment]
> So every time I clone my empty repo from github and try to do bunx create hono I get a message saying the file is not empty despite there only being the invisible .git file and if i select yes it gives an error - " export default "/tmp/bunx-1000-create-hono@latest/node_modules/create-hono/bin"; ^ error: could not find commit hash for main code: "MISSING_REF"
> 
> ```
>   at new DegitError (/tmp/bunx-1000-create-hono@latest/node_modules/create-hono/bin:17108:9)
>   at /tmp/bunx-1000-create-hono@latest/node_modules/create-hono/bin:17473:17"
> ```
> 
> and deletes my folder . How can I overcome this

So it is working using npm create hono@latest not bun

**Reply by 26thavenue** [reply]
Is it supported for bun ?

**Comment by 26thavenue** [comment, marked as answer]
> > So every time I clone my empty repo from github and try to do bunx create hono I get a message saying the file is not empty despite there only being the invisible .git file and if i select yes it gives an error - " export default "/tmp/bunx-1000-create-hono@latest/node_modules/create-hono/bin"; ^ error: could not find commit hash for main code: "MISSING_REF"
> > ```
> >   at new DegitError (/tmp/bunx-1000-create-hono@latest/node_modules/create-hono/bin:17108:9)
> >   at /tmp/bunx-1000-create-hono@latest/node_modules/create-hono/bin:17473:17"
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > and deletes my folder . How can I overcome this
> 
> So it is working using npm create hono@latest not bun


It was because I didnt use@latest when creating it with bun , but is this supposed to be a normal behaviour

**Reply by NicoPlyley** [reply]
Are you doing `bunx create-hono`?
